### ISSUE TITLE
Cant search models with Short names 1-4 symbols

### ISSUE THREAD

**Issue by hokagegaara** [Frostshake/WMVx#74]
Cant search models with short names who have >5 symbols(1-4symbols)

Example: lili
![изображение](https://github.com/user-attachments/assets/6993d616-ab62-4547-9053-ce7ce265127d)
![изображение](https://github.com/user-attachments/assets/caf97dd0-a7c8-438d-a207-8e7b5e3d7e48)

**Comment by hokagegaara** [comment]
Maybe add setting and search button so that.
Setting: Search on button press or keypress Enter(And not like now, when you enter text, it immediately searches).
Adding this feature optimizes the search for low-end hardware

**Comment by hokagegaara** [comment]
Or you are looking for a model with a specific short name.
Ice, Key, 6hu_(Models from WoD in human style) and other.

**Comment by Frostshake** [comment]
Known problem (related to #55).
The current search length limitation is due to the UI becoming slow with a large result set - I'm unsure how this can be overcome.

**Comment by hokagegaara** [comment]
> Known problem (related to #55). The current search length limitation is due to the UI becoming slow with a large result set - I'm unsure how this can be overcome.
Yes, just add a forced search option "button"(Or Press Enter) or setting for good PC.

Also maybe for "Scene/Component":
Add 3 button's "remove all components", "only 1 component yes/no"(When adding a new model it deletes the previous one), and "add the selected model directly to the component"
If you Press on Keyboard "Backspace" or "Delete" buttons on "component" he will be removed/deleted.
